src/navsat_transform.cpp

```diff
@@ -113,8 +113,10 @@ NavSatTransform::NavSatTransform(const rclcpp::NodeOptions & options)
   from_ll_srv_ = this->create_service<robot_localization::srv::FromLL>(
     "fromLL", std::bind(&NavSatTransform::fromLLCallback, this, _1, _2));
 
-  std::vector<double> datum_vals;
-  if (use_manual_datum_ && this->get_parameter("datum", datum_vals)) {
+  if (use_manual_datum_) {
+    std::vector<double> datum_vals(3, 0);
```

Comment:
Lets leave this as the default constructor. By doing this, you create and destroy a few objects that aren't required.  The rest looks good.

Reply:
I reverted the vector to leave on default constructor.